src/components/tables/RowActions/Shared/Confirmation.tsx

```diff
@@ -1,7 +1,10 @@
 import { List, ListItem, Typography } from '@mui/material';
+import { RowConfirmation } from '../types';
 import NestedListItem from './NestedListItem';
 import { RowActionConfirmationProps } from './types';
 
+// TODO (typing) - eventually we should remove supporting passing in string arrays
+//  so that we always have an `id` and `message` to use.
```

Comment:
Requesting a proper `id` AND `message` makes this much more resilient but a bigger change than needed for the prod issue.